src/sections/dataset/dataset-files/DatasetFiles.tsx

```diff
@@ -1,47 +1,65 @@
-import { useFilesTable } from './files-table/useFilesTable'
 import { FileRepository } from '../../../files/domain/repositories/FileRepository'
-import { useFiles } from './useFiles'
-import { useEffect, useState } from 'react'
+import { useState } from 'react'
 import { FilesTable } from './files-table/FilesTable'
-import { SpinnerSymbol } from './files-table/spinner-symbol/SpinnerSymbol'
-import { FileCriteriaInputs } from './file-criteria-inputs/FileCriteriaInputs'
-import { FileCriteria } from '../../../files/domain/models/FileCriteria'
+import { FileCriteriaControls } from './file-criteria-controls/FileCriteriaControls'
+import { FileAccessOption, FileCriteria, FileTag } from '../../../files/domain/models/FileCriteria'
+import { FilesCountInfo } from '../../../files/domain/models/FilesCountInfo'
+import { FileType } from '../../../files/domain/models/File'
 
 interface DatasetFilesProps {
   filesRepository: FileRepository
   datasetPersistentId: string
   datasetVersion?: string
 }
 
+const MINIMUM_FILES_TO_SHOW_CRITERIA_INPUTS = 2
+const filesCountInfo: FilesCountInfo = {
+  total: 222,
+  perFileType: [
+    {
+      type: new FileType('text'),
+      count: 5
+    },
+    {
+      type: new FileType('image'),
+      count: 485
+    }
+  ],
+  perAccess: [
+    { access: FileAccessOption.PUBLIC, count: 222 },
+    { access: FileAccessOption.RESTRICTED, count: 10 }
+  ],
+  perFileTag: [
+    { tag: new FileTag('document'), count: 5 },
+    { tag: new FileTag('code'), count: 10 }
+  ]
+} // TODO (filesCountInfo) - Get from use case, pending to be discussed if this is going to have its own use case or not
```

Comment:
Let's discuss about this to decide how to implement the use case logic before accepting this PR.

Reply:
My personal preference is getting the filesCountInfo from the getDataset use case. Just because I think that having another use case will increase the time for opening a dataset view. But let me know what you think so I can implement it